lua/entities/gmod_wire_expression2/base/tokenizer.lua

```diff
@@ -145,44 +145,35 @@ function Tokenizer:NextSymbol()
 
 	elseif self:NextPattern("^[a-z#][a-zA-Z0-9_]*") then
 		-- keywords/functions
-		if self.tokendata == "if" then
-			tokenname = "if"
-		elseif self.tokendata == "elseif" then
-			tokenname = "eif"
-		elseif self.tokendata == "else" then
-			tokenname = "els"
-		elseif self.tokendata == "local" then
-			tokenname = "loc"
-		elseif self.tokendata == "while" then
-			tokenname = "whl"
-		elseif self.tokendata == "for" then
-			tokenname = "for"
-		elseif self.tokendata == "break" then
-			tokenname = "brk"
-		elseif self.tokendata == "continue" then
-			tokenname = "cnt"
-		elseif self.tokendata == "switch" then
-			tokenname = "swh"
-		elseif self.tokendata == "case" then
-			tokenname = "case"
-		elseif self.tokendata == "default" then
-			tokenname = "default"
-		elseif self.tokendata == "foreach" then
-			tokenname = "fea"
-		elseif self.tokendata == "function" then
-			tokenname = "func"
-		elseif self.tokendata == "return" then
-			tokenname = "ret"
-		elseif self.tokendata == "void" then
-			tokenname = "void"
-		elseif self.tokendata == "#include" then
-			tokenname = "inclu"
+		local keywords_and_tokens = {
```

Comment:
Can this table please live in `e2lib.lua` next to `E2Lib.optable`? It'd make sense in future to be able to reuse this in the E2 text editor, plus we already have a table of operators.

Reply:
I told him in the [comment](https://github.com/wiremod/wire/pull/1916#issuecomment-489397618) already, but yeah, this should be moved outside.